Makefile

```diff
@@ -10,9 +10,9 @@ CFLAGS = -Wall -DENABLE_BPF -DHAVE_PF_RING -DHAVE_PF_RING_ZC -DTAPDANCE_USE_PF_R
 PROTO_RS_PATH=src/signalling.rs
 EXE_DIR=./bin
 
-all: rust libtd conjure app registration-server ${PROTO_RS_PATH}
+all: rust libtd conjure app registration-server registration-tracker ${PROTO_RS_PATH}
```

Comment:
Is registration-tracker something we want built when we build the station? I would think this is more of a utility, so it wouldn't require building every time we make a production station, more for when we want to debug.